src/main/java/task/TaskList.java

```diff
@@ -0,0 +1,74 @@
+package task;
+
+import java.util.ArrayList;
+
+/**
+ * Abstraction for data structure and operations used to maintain to do list.
+ */
+public class TaskList {
+    private ArrayList<Task> records = new ArrayList<>();
+
+    // default constructor
+    public void add(Task task) {
+        records.add(task);
+        System.out.println("B: " + task.toString() + "has been added");
```

Comment:
I recommend adding a whitespace before "has been added". I notice that this might cause an issue whereby it becomes "workhas been added" unless the user themselves add a whitespace. 
```suggestion
        System.out.println("B: " + task.toString() + " has been added");
```